ibm_i/setup.py

```diff
@@ -27,7 +27,22 @@ def get_dependencies():
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base>=21.0.0'
+def parse_pyproject_array(name):
```

Comment:
Since this integration is Python 3 only, do we still need the setup.py file?

Reply:
good catch, addressed!